packages/connectivity_plus/connectivity_plus/example/integration_test/connectivity_plus_test.dart

```diff
@@ -25,11 +33,11 @@ void main() {
       expect(result, isNotNull);
     });
 
-    testWidgets('connectivity on Android emulator should be wifi',
+    testWidgets('connectivity on ${getPlatformName()} should be wifi',
```

Comment:
I'd rather not do this, and instead just name it `Connectivity should be wifi`

Reply:
Or maybe just write one test per platform, it is ok if tests are duplicated IMO

Reply:
oh okay sure.